README.md

```diff
@@ -73,6 +73,9 @@ The CLI currently supports the following commands (and sub-commands):
   - `ruby`:                 Push (upload) a new Ruby package upstream.
   - `terraform`:            Push (upload) a new Terraform package upstream.
   - `vagrant`:              Push (upload) a new Vagrant package upstream.
+- `quarantine`|`qu`:      Manage quarantined packages in a repository.
```

Comment:
Not 100% sure on the shorthand here...I get it because quarantine is a long word that I spell incorrectly very regularly 😂  But the `qu` also forms part of `quota` below. `quar` maybe?

Reply:
ah, yeah, that makes sense :sweat_smile: 

Might want to skip the shorthand completly tho: https://www.urbandictionary.com/define.php?term=quar

Reply:
Haha, yes! 😂